packages/esbuild-transpiler/package.json

```
@@ -47,20 +63,23 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/honojs/middleware.git"
+    "url": "git+https://github.com/honojs/middleware.git",
+    "directory": "esbuild-transpiler"
```

This should be `packages/esbuild-transpiler`.

---

Nice pickup, thanks!

```suggestion
    "directory": "packages/esbuild-transpiler"
```

I did seperate PRs hoping it would be easier to review and pickup small things like this. Sorry there are so many! 😅

---

No problem!